app/controllers/pages_controller.rb

```diff
@@ -11,5 +11,10 @@ def dashboard
     @data[:maintenances] = current_user.maintenances.most_recent(5)
     @data[:mileages] = current_user.mileages.most_recent(5).decorate
     @data[:reminders] = current_user.reminders.most_recent(5)
+    @data[:closest_service] = find_nearby['car_repair']
```

Comment:
Why ``` find_nearby['car_repair'] ```? Shouldn't be it ``` find_nearby ```

Reply:
Yep, it's not too pretty. I prepared find_nearby service to take few "places" from PLACES API (car_dealer, car_rental, car_wash, etc.) and returns it in hash. So to add new place to view we can simply add new attribute to the controller.

Reply:
There is above an unused ```  @data[:closest_service]  ``` declaration :)
Remove this.